app/models/service_provider_identity.rb

```diff
@@ -15,6 +15,7 @@ class ServiceProviderIdentity < ApplicationRecord
   # rubocop:enable Rails/InverseOf
 
   scope :not_deleted, -> { where(deleted_at: nil) }
+  scope :consented, -> { where.not(last_consented_at: nil) }
```

Comment:
I'm generally against it since it can be kind of surprising, but is this a situation for [default_scope](https://api.rubyonrails.org/v7.0.4/classes/ActiveRecord/Scoping/Default/ClassMethods.html#method-i-default_scope) that defaults to consented identities?

The risks involved are not small, and I don't think we should commit to using it long-term, but the column is a big shift in the assumptions we can make about an identity.

Reply:
This was actually my first instinct, but a previous project I was on ran into a whole lot of trouble that ultimately convinced me it was evil.

The idea of a _short-term_ default scope is a little tempting, though, especially with some urgency around this. What say ye, @zachmargolis?

Reply:
I've been bitten by `default_scope`... I think our uses cases here are mixed enough that we don't want to use that here

Reply:
I don't have any strong opinions on "short-term" but will say that I feel like saying something is short term doesn't guarantee we'll actually remove it quickly 😂

Reply:
Yeah, I'm mostly looking for safeguards to not treat users that have not consented as though they have